logpyle/__init__.py

```diff
@@ -1260,8 +1260,12 @@ class LogUpdateDuration(PostLogQuantity):
     def __init__(self, mgr: LogManager, name: str = "t_log") -> None:
         LogQuantity.__init__(self, name, "s", "Time spent updating the log")
         self.log_manager = mgr
+        self.done_hwm = False
 
     def __call__(self) -> float:
+        if not self.done_hwm:
+            self.done_hwm = True
+            add_hwm(self.log_manager)
```

Comment:
Not the greatest place for this, but I couldn't find a better solution. In particular, `LogManager.close()` won't work since this is run from e.g. runalyzer-gather as well.